wizeline/wize-q-remix
# feat: restrict questions list (homepage)

#### What does this PR do?
- Add a where a clause in the controller of the list questions. 
- Show the questions in the last 3 months.

#### Where should the reviewer start?
- Check the createDateRange:  app/utils/backend/dateUtils.j
      for the lastDate add an extra day, because the where clause not include the last date. 
- Review run the tests `npm run test:integration`

#### How should this be manually tested?
1. Login
2. Verify that you can see only the questions in the last 3 months. 
3. Verify that you can get information applying the filter, but just for the last 3 months. 

#### What are the relevant tickets?
(Link to issues, related PR, JIRA issues, etc.)
Closes #105

#### Screenshots
[tests]

<img width="1046" alt="Screen Shot 2022-11-29 at 10 48 01" src="https://user-images.githubusercontent.com/97203617/204591134-e1b1fb5a-2bd9-4687-8881-d8416e686cd2.png">


https://user-images.githubusercontent.com/97203617/204623632-1c745506-0147-4d7b-af36-babd66ef8db2.mov

###Extra:

- For a question search, you can see questions older than 3 months. 
- If you have a question pinned more than 3 months ago, you can still see it on the home page.
